services/galley/src/Galley/Data.hs

```diff
@@ -429,14 +432,20 @@ createOne2OneConversation a b name ti = do
             addPrepQuery Cql.insertConv (conv, One2OneConv, a', privateOnly, privateRole, fromRange <$> name, Just tid, Nothing)
             addPrepQuery Cql.insertTeamConv (tid, conv, False)
     mems <- snd <$> addMembersUnchecked now conv a' (list1 a' [b'])
-    return $ newConv conv One2OneConv a' (toList mems) [PrivateAccess] privateRole name ti Nothing
+    return $ newConv conv One2OneConv a' (toList mems) [PrivateAccess] privateRole name ti Nothing Nothing
```

Comment:
Why is there no possibility of setting the readReceiptMode on conversation creation (i.e. why hardcode `Nothing` here?)?

Reply:
Changed it now, but only for group convs.